darts/tests/models/forecasting/test_backtesting.py

```diff
@@ -100,6 +100,26 @@ def test_backtest_forecasting(self):
         )
         self.assertEqual(score, 1.0)
 
+        # very large train length should not affect the backtest
+        score = NaiveDrift().backtest(
```

Comment:
Thanks for the unit tests!
Maybe we could add one that checks that the backtest accuracy remains OK for of one of the model when a `train_length` is applied?